model_info/nlwiki.wp10.md

```diff
@@ -34,51 +34,51 @@ Model Information:
 	match_rate (micro=0.2, macro=0.2):
 		    E      D      C      B      A
 		-----  -----  -----  -----  -----
-		0.147  0.256  0.204  0.196  0.198
+		0.133  0.271  0.201  0.198  0.196
 	filter_rate (micro=0.8, macro=0.8):
 		    E      D      C      B      A
 		-----  -----  -----  -----  -----
-		0.853  0.744  0.796  0.804  0.802
-	recall (micro=0.745, macro=0.745):
-		    E      D      C      B     A
-		-----  -----  -----  -----  ----
-		0.727  0.812  0.876  0.621  0.69
-	!recall (micro=0.936, macro=0.936):
-		    E      D      C      B      A
-		-----  -----  -----  -----  -----
-		0.998  0.883  0.964  0.911  0.925
-	precision (micro=0.764, macro=0.764):
+		0.867  0.729  0.799  0.802  0.804
+	recall (micro=0.734, macro=0.734):
 		    E      D     C      B      A
 		-----  -----  ----  -----  -----
-		0.992  0.635  0.86  0.634  0.697
-	!precision (micro=0.937, macro=0.937):
-		    E     D      C      B      A
-		-----  ----  -----  -----  -----
-		0.936  0.95  0.969  0.906  0.923
-	f1 (micro=0.748, macro=0.748):
+		0.661  0.815  0.87  0.633  0.693
+	!recall (micro=0.934, macro=0.934):
 		    E      D      C      B      A
 		-----  -----  -----  -----  -----
-		0.839  0.713  0.868  0.628  0.693
-	!f1 (micro=0.936, macro=0.936):
+		0.998  0.865  0.966  0.911  0.928
+	precision (micro=0.76, macro=0.76):
 		    E      D      C      B      A
 		-----  -----  -----  -----  -----
-		0.966  0.915  0.967  0.908  0.924
-	accuracy (micro=0.898, macro=0.898):
+		0.991  0.602  0.864  0.639  0.707
+	!precision (micro=0.934, macro=0.934):
 		    E      D      C      B      A
 		-----  -----  -----  -----  -----
-		0.944  0.869  0.947  0.853  0.878
-	fpr (micro=0.064, macro=0.064):
+		0.922  0.949  0.967  0.909  0.924
+	f1 (micro=0.738, macro=0.738):
+		    E      D      C      B    A
+		-----  -----  -----  -----  ---
+		0.793  0.692  0.867  0.636  0.7
+	!f1 (micro=0.933, macro=0.933):
+		    E      D      C     B      A
+		-----  -----  -----  ----  -----
+		0.959  0.905  0.967  0.91  0.926
+	accuracy (micro=0.894, macro=0.894):
```

Comment:
Looks like we lost a bit of accuracy.  That could be related to random chance.   The difference is small.

Reply:
Yes I thought so too! Do you think this changes are required?